dummyData.sql

```diff
@@ -0,0 +1,79 @@
+--USERS:
```

Comment:
Love it! Makes it easy for someone to get up and running on your project, and for you to have a demo-able app ready and committed